src/pancam.py

```diff
@@ -0,0 +1,22 @@
+from follower import Follower
```

Comment:
Overall, This shouldn't be a separate file (or a separate class). Seeking should be a function inside of the follower class that accompanies the rest of the follower's functionalities.